HaulageApplication/Haulage/BaseClasses/Accounting/Customer.cs

```diff
@@ -36,4 +53,22 @@ public void manageBilling()
         }
 
     }
+
+// = ACCOUNT = START ================================================================================================================ //
+    public class Account
+    {
+        public Guid AccountId { get; set; }
+        public string BillingDetails { get; set; }
+        public string ContactDetails { get; set; }
+
+        public Account(Guid accountId, string billingDetails, string contactDetails)
+        {
+            AccountId = accountId;
```

Comment:
should accountID be an integer instead? so that it is easier to put into the database as an INT in the future

Reply:
Hey Shaun, Interresting theory, however:
-GUID Pros-
Unique across every table, every database and every server
Allows easy merging of records from different databases
Allows easy distribution of databases across multiple servers
You can generate IDs anywhere, instead of having to roundtrip to the database, unless partial sequentiality is needed (i.e. with newsequentialid())
Most replication scenarios require GUID columns anyway
-GUID Cons-
It is a whopping 4 times larger than the traditional 4-byte index value; this can have serious performance and storage implications if you're not careful
Cumbersome to debug (where userid='{BAE7DF4-DDF-3RG-5TY3E3RF456AS10}')
The generated GUIDs should be partially sequential for best performance (eg, newsequentialid() on SQL Server 2005+) and to enable use of clustered indexes